### PR TITLE
change shape for mearec

### PR DESCRIPTION
**Pull request by samuelgarcia**

### CODE REVIEW

File: spikeextractors/extractors/mearecextractors/mearecextractors.py [hunk 1]
```diff
@@ -4,12 +4,17 @@
...
 
 try:
     import MEArec as mr
     import neo
     import quantities as pq
-    HAVE_MREX = True
+    if StrictVersion(mr.__version__) >= '1.5.0':
+        HAVE_MREX = True
+    else:
+        print('MEArec is too old, you must do : pip install --upgrade MEArec please')
```
Comment (alejoe91) [inline spikeextractors/extractors/mearecextractors/mearecextractors.py:16]: @samuelgarcia: "MEArec version requires an update (>=1.5). Please upgrade with 'pip install --upgrade MEArec'"

File: spikeextractors/extractors/mearecextractors/mearecextractors.py [hunk 2]
```diff
@@ -73,12 +78,12 @@ def get_sampling_frequency(self):
     def get_traces(self, channel_ids=None, start_frame=None, end_frame=None):
         if np.any(np.diff(channel_ids) < 0):
             sorted_idx = np.argsort(channel_ids)
-            recordings = self._recordings[np.sort(channel_ids), start_frame:end_frame]
+            recordings = self._recordings[start_frame:end_frame, np.sort(channel_ids)]
```
Comment (alejoe91) [inline spikeextractors/extractors/mearecextractors/mearecextractors.py:81]: @samuelgarcia you need to transpose this

File: spikeextractors/extractors/mearecextractors/mearecextractors.py [hunk 3]
```diff
@@ -73,12 +78,12 @@ def get_sampling_frequency(self):
...
             return recordings[sorted_idx]
         else:
             if sorted(channel_ids) == channel_ids and np.all(np.diff(channel_ids) == 1):
                 channel_ids = slice(channel_ids[0], channel_ids[0] + len(channel_ids))
-            return self._recordings[channel_ids, start_frame:end_frame]
+            return self._recordings[start_frame:end_frame, channel_ids]
```
Comment (alejoe91) [inline spikeextractors/extractors/mearecextractors/mearecextractors.py:86]: @samuelgarcia you need to transpose this

(until we change internally get_traces shape)

**Review by alejoe91 — CHANGES_REQUESTED** [review]

File: spikeextractors/extractors/mearecextractors/mearecextractors.py (hunk 2 above)
Comment (samuelgarcia) [reply spikeextractors/extractors/mearecextractors/mearecextractors.py:81]: It was done I was sure I push it.

Comment (samuelgarcia) [reply spikeextractors/extractors/mearecextractors/mearecextractors.py:81]: In fact, I have push it on the main repo. Oups.
i need to delete that branch.

**Comment by samuelgarcia** [comment]
Alessio : you should release mearec and then we could merge this soon.

**Comment by samuelgarcia** [comment]
I think we can merge no ?

File: spikeextractors/extractors/mearecextractors/mearecextractors.py (hunk 2 above)
Comment (alejoe91) [reply spikeextractors/extractors/mearecextractors/mearecextractors.py:81]: @samuelgarcia this still needs a transpose

**Review by alejoe91 — APPROVED** [review]

**Pull request merged**